README.md

```diff
@@ -1,86 +1,173 @@
+2016-02-25 16:06:38,298 - [INFO] - main.py - Action/Mode Selected is: run
+2016-02-25 16:06:38,299 - [INFO] - base.py - Unpacking image: projectatomic/helloapache to /var/lib/atomicapp/projectatomic-helloapache-7c32c1632a7b
+2016-02-25 16:06:41,904 - [INFO] - container.py - Skipping pulling Docker image: projectatomic/helloapache
+2016-02-25 16:06:41,904 - [INFO] - container.py - Extracting nulecule data from image: projectatomic/helloapache to /var/lib/atomicapp/projectatomic-helloapache-7c32c1632a7b
+20af2e6e33d10d26aa98d6e63c70de5fd55bfe14b9cc782e1312afe441ef7130
+2016-02-25 16:06:42,231 - [INFO] - docker.py - Deploying to provider: Docker
+5d6938439d50c21251507b26c73f5e65f102f2b99e183002ef2ec21414c4ee78
+
+Your application resides in /var/lib/atomicapp/projectatomic-helloapache-7c32c1632a7b
+Please use this directory for managing your application
+
+▶ docker ps
+CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                NAMES
+5d6938439d50        centos/httpd        "/run-httpd.sh"          3 seconds ago       Up 1 seconds        0.0.0.0:80->80/tcp   default_centos-httpd_ec75a2fe2a50
 ```
 
-### Building for containerized execution
+__Runnning Apache on Kubernetes:__
+```sh
+▶ sudo atomicapp run --provider=kubernetes projectatomic/helloapache
+2016-02-25 15:03:04,341 - [INFO] - main.py - Action/Mode Selected is: run
+2016-02-25 15:03:04,343 - [INFO] - base.py - Unpacking image: projectatomic/helloapache to /var/lib/atomicapp/projectatomic-helloapache-c0dd79b5e757
+2016-02-25 15:03:07,983 - [INFO] - container.py - Skipping pulling Docker image: projectatomic/helloapache
+2016-02-25 15:03:07,984 - [INFO] - container.py - Extracting nulecule data from image: projectatomic/helloapache to /var/lib/atomicapp/projectatomic-helloapache-c0dd79b5e757
+886e10a3244f982f3302ab9058ab7b377c6f83e2cf63f001e1ba011358d0b471
+2016-02-25 15:03:08,332 - [INFO] - kubernetes.py - Using namespace default
+2016-02-25 15:03:08,332 - [INFO] - kubernetes.py - trying kubectl at /usr/bin/kubectl
+2016-02-25 15:03:08,332 - [INFO] - kubernetes.py - trying kubectl at /usr/local/bin/kubectl
+2016-02-25 15:03:08,332 - [INFO] - kubernetes.py - found kubectl at /usr/local/bin/kubectl
+2016-02-25 15:03:08,332 - [INFO] - kubernetes.py - Deploying to Kubernetes
+...
+
+Your application resides in /var/lib/atomicapp/projectatomic-helloapache-c0dd79b5e757
+Please use this directory for managing your application
+
+▶ kubectl get po
+NAME                   READY     STATUS    RESTARTS   AGE
+helloapache            1/1       Running   0          2m
+k8s-etcd-127.0.0.1     1/1       Running   0          1d
+k8s-master-127.0.0.1   4/4       Running   0          1d
+k8s-proxy-127.0.0.1    1/1       Running   0          1d
 ```
-docker build -t [TAG] .
+
+__Fetch, edit and run Apache on Kubernetes:__
+```sh
+▶ sudo atomicapp fetch --destination helloapache projectatomic/helloapache
```

Comment:
can we rename the destination to be more obvious that it is a local folder. Maybe something like:

```
mkdir ./localdir
sudo atomicapp fetch projectatomic/helloapache  --destination ./localdir/
```